docs/maintainer-guide/pullrequests.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Reviewing Pull Requests
+layout: doc
+---
+
+# Reviewing Pull Requests
+
+Pull requests are submitted frequently and represent our best opportunity to interact with the community. As such, it's important that pull requests are well-reviewed before being merged and that interactions on pull requests are positive.
+
+## Who Can Review Pull Requests?
+
+Anyone, both team members and the public, may leave comments on pull requests.
+
+## Who Can Approve Pull Requests?
+
+Only project reviewers or admins may use the "Merge" button to merge in changes.
+
+## Reviewing a Pull Request
+
+When a pull request is opened, follow these steps:
+
+1. Has the submitter signed a CLA? If not, please ask them to do so.
+1. Is the commit message summary in the correct format? Double-check that the tag ("Fix:", "New:", etc.) is correct based on the issue. Documentation-only pull requests do not require an issue.
```

Comment:
Should probably mention that bot will check for those and leave a message, if it finds problems.